client/src/components/StepWizard/WizardContainer.js

```diff
@@ -1,25 +1,69 @@
 import styled from "styled-components";
 
-import { mq } from "../../constants/theme";
+import { mq, theme } from "../../constants/theme";
 
-const { desktop } = mq;
+const { desktop, tablet } = mq;
 
 const marginTop = "100px";
 
 const WizardContainer = styled.div`
+font-family: ${theme.typography.font.family.display}
   display: flex;
   flex-flow: row wrap;
-  margin: ${marginTop} 0 0 0;
-  max-width: calc(100vw - 50px);
-  height: 100vh;
+  margin: 5rem 0 0 0;
+  width:100%;
   max-height: calc(100vh - 153px);
   justify-content: center;
 
-  @media screen and (min-width: ${desktop.small.minWidth}) {
+  .title{
+    font-weight: bold;
+    font-size: 1.6rem;
+    margin: 1rem 0;
+  }
+
+  .sub-title{
+    font-weight: 200;
+    font-size: 1.4rem;
+  }
+
+  .content-wrapper{
+    width:100%;
+    margin: 3rem 0;
+    font-weight: regular;
+    font-size: 1.4rem;
+  }
+
+  @media screen and (min-width: ${tablet.narrow.minWidth}) {
+    display: flex;
+    justify-content: center;
     margin: ${marginTop} auto;
-    width: 100%;
-    max-width: 650px;
+    max-width: 850px;
+
+    .title{
```

Comment:
We have <Heading> & <TextLabel> components so you don't have to create new styles for them

Reply:
got rid of that. please take a look to the 
client/src/components/SymptomsCheck/SymptomsCheckStyles.js